lib/internal/fs/rimraf.js

```diff
@@ -249,7 +249,7 @@ function _rmdirSync(path, options, originalErr) {
 
       for (let i = 1; i <= tries; i++) {
         try {
-          return rmdirSync(path, options);
+          return rmdirSync(path, { ...options, recursive: false });
```

Comment:
I think you can just call `rmdirSync(path)` here. If `recursive` is `false`, then `maxRetries` and `retryDelay` are ignored anyway.